recipes/dttlib/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "dttlib" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/software/cr_tools/dttlib/-/archive/{{ version }}/{{ version }}.tar.gz
+  sha256: 3529939797ec31d32afa30f94633c225988dbc0548b6ad1b882c85da78ea6215 
+
+build:
+  skip: true  # [win or osx]
+  error_overlinking: true
+  error_overdepending: true
+  number: 0
+
+requirements:
+  build:
+   - {{ compiler('c') }}
+   - {{ stdlib("c") }}
+   - {{ compiler('cxx') }}
+   - python
+   - clang
+   - fftw 
+   - gcc
+   - gds-sigp 
+   - glib
+   - libcds 
+   - libclang
```

Comment:
Once you've specified `clang` as you compiler as described, you should hopefully not need this dependency any more.